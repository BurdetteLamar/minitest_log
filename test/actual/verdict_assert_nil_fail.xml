<log>
  <summary verdicts='2' failures='2' errors='0'/>
  <section name='Test' method='verdict_assert_nil?' type='fail'>
    <section name='Args=[0]'>
      <verdict method='verdict_assert_nil?' outcome='failed' id='0' message='Message 0'>
        <actual class='Integer' value='0'/>
        <exception class='Minitest::Assertion' message='Expected 0 to be nil.'>
          <backtrace/>
        </exception>
      </verdict>
    </section>
    <section name='Args=[false]'>
      <verdict method='verdict_assert_nil?' outcome='failed' id='1' message='Message 1'>
        <actual class='FalseClass' value='false'/>
        <exception class='Minitest::Assertion' message='Expected false to be nil.'>
          <backtrace/>
        </exception>
      </verdict>
    </section>
  </section>
</log>
