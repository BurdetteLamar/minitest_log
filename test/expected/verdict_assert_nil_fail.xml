<log>
  <summary verdicts='3' failures='2' errors='0'/>
  <section name='Test' method='verdict_assert_nil?' type='fail'>
    <section name='Args=[0]'>
      <verdict method='verdict_assert_nil?' outcome='failed' id='0' message='Message 0'>
        <act_value>
          0
        </act_value>
        <exception class='Minitest::Assertion'>
          <message>
            Expected 0 to be nil.
          </message>
          <backtrace/>
        </exception>
      </verdict>
    </section>
    <section name='Args=[false]'>
      <verdict method='verdict_assert_nil?' outcome='failed' id='1' message='Message 1'>
        <act_value>
          false
        </act_value>
        <exception class='Minitest::Assertion'>
          <message>
            Expected false to be nil.
          </message>
          <backtrace/>
        </exception>
      </verdict>
    </section>
  </section>
  <verdict method='verdict_assert_equal?' outcome='passed' id='__error_count__'>
    <exp_value>
      0
    </exp_value>
    <act_value>
      0
    </act_value>
  </verdict>
</log>
